src/workloads/docs/Loader.yml

```diff
@@ -0,0 +1,79 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+
+# More threads requires a larger connection pool.
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 200
+
+Actors:
+  - Name: Loader
+    Type: Loader
+    Threads: 10
+    Phases:
+      - Repeat: 1
+        Database: cold
+        CollectionCount: 6
+        Threads: 1
+        DocumentCount: 200000
+        BatchSize: 100000
+        Document:
+          a: {^RandomString: { length: 100 }}
+      - Repeat: 1
```

Comment:
I don't understand the point of his example outside of the last loader phase and maybe the first. What are the hot phases doing?